src/OneSignalMessage.php

```diff
@@ -85,7 +60,11 @@ public function icon($value)
      */
     public function subject($value)
     {
-        $this->subject = $value;
+        if (is_array($value)) {
+            $this->setParameter('headings', $value);
+        } else {
+            $this->setParameter('headings', ['en' => $value]);
+        }
```

Comment:
Same here as with  body - however if going with the array check version, you might want to introduce a `parseLocale()` method to remove the duplicate code...

```php
protected function parseLocale($value)
{
    return is_array($value) ? $value : ['en' => $value];
}

// then subject / body etc with localisation could just be

public function subject()
{
    $this->setParameter('headings', $this->parseLocale($value));
}
```

Reply:
Yeah thats a better solution!